base-system/usrroot/usr/lib/hsync/libhapply.so

```diff
@@ -47,6 +47,7 @@ apply_global_directives() {
 	set_timezone
 	set_keyboards
 	set_current_keyboard
+	set_scheduled_shutdown
```

Comment:
Let's condition the shutdown to the clock being NTP sync. Otherwise this could lead computers to shutdown in an unwanted moment if the hardware clock is out of sync. 
This is important to also be updated on the documentation.

Reply:
Updating that